docs/cli/etcd-snapshot.md

```diff
@@ -72,19 +113,18 @@ k3s etcd-snapshot save \
   --s3-secret-key=<S3-SECRET-KEY>
 ```
 
-To perform an on-demand etcd snapshot restore from S3, first make sure that K3s isn't running. Then run the following commands:
-
 ```bash
-k3s server \
-  --cluster-init \
-  --cluster-reset \
-  --etcd-s3 \
-  --cluster-reset-restore-path=<SNAPSHOT-NAME> \
-  --etcd-s3-bucket=<S3-BUCKET-NAME> \
-  --etcd-s3-access-key=<S3-ACCESS-KEY> \
-  --etcd-s3-secret-key=<S3-SECRET-KEY>
+k3s etcd-snapshot delete          \
+  --s3                            \
+  --s3-bucket=<S3-BUCKET-NAME>    \
+  --s3-access-key=<S3-ACCESS-KEY> \
+  --s3-secret-key=<S3-SECRET-KEY> \
+  <SNAPSHOT-NAME>
 ```
 
+As snapshots are replicated in S3, these commands will create and delete two snapshots: one stored in the file system and one in S3.
+
```

Comment:
This line is confusing. Are you saying it creates 2 copies of the snapshot, upload one and deletes the other?

Reply:
Yeah, this is confusing as written. I would probably change this to an example command with output using the `shell-session` block style, as we do for the `$ kubectl describe etcdsnapshotfile` examples below.